fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/ext/WaitNotNull.kt

```diff
@@ -17,4 +18,8 @@ import org.mozilla.fenix.helpers.TestAssetHelper
 fun UiDevice.waitNotNull(
     searchCondition: SearchCondition<*>,
     waitTime: Long = TestAssetHelper.waitingTime,
-) = assertNotNull(wait(searchCondition, waitTime))
+) {
+    Log.d("TestLog", "Wait not null: $searchCondition")
```

Comment:
Should modify your log tag to match the rest of the logging statements and change it to `mozTestLog` here and the logging statement below

Reply:
Yes, I missed that detail. Thanks for catching it.